src/common/snippets/include/snippets/lowered/linear_ir.hpp

```diff
@@ -46,6 +54,7 @@ class LinearIR {
     const container& get_ops() const {return m_expressions; }
     const io_container& get_IO_ops() const {return m_io_expressions; }
     Config get_config() {return m_config; }
+    void set_loop_depth(size_t loop_depth) { m_config.m_loop_depth = loop_depth; }
```

Comment:
Thanks for the suggestion, but `move` makes no sense for fundamental types, since they have no move constructors. Please refer to [here](https://stackoverflow.com/a/14679830) or [here](https://stackoverflow.com/a/5113385) for more details.